end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -159,6 +173,8 @@ The *mzML* files with the raw MS data are located within the *data/mzML* folder
 of this repository. [ideally, they should be added and then downloaded from
 MetaboLight].
 
+Note that not different equipments will generate different files extensions, so a conversion step might be needed before, although it doesnt apply to this dataset. Several tools can be used to convert files to .mzML format, such as ProteoWizard`s MSConvert
```

Comment:
good point. we should add a sentence on file conversion - or mention the different backends we have that would/could also support the original raw files from the vendors (at least for Thermo and Bruker TimsTof).